src/Features/Core/Portable/FeaturesResources.resx

```diff
@@ -1334,4 +1334,7 @@ This version used in: {2}</value>
   <data name="indexer_" xml:space="preserve">
     <value>indexer</value>
   </data>
-</root>
+  <data name="Regex_issue_0" xml:space="preserve">
+    <value>Regex issue: {0}</value>
```

Comment:
Note: localization seems to have changed a lot.  There are a lot of these new .xlf files.  They added a lot of bloat to this PR.  But i wasn't sure what else to do.  Let me know if i need to do something different here.